test.c

```diff
@@ -77,6 +79,51 @@ static long long usec(void) {
 #define assert(e) (void)(e)
 #endif
 
+/* Attempt to get an INFO field */
+sds get_info_value(redisContext *c, const char *field, size_t len) {
+    redisReply *reply;
+    sds val;
+    char *s, *e;
+
+    reply = redisCommand(c, "INFO");
+    if (reply == NULL || c->err) {
+        fprintf(stderr, "Error:  Can't execute INFO, aborting!\n");
+        exit(1);
+    }
+
+    if ((s = strstr(reply->str, field)) == NULL || *(s+len) != ':' ||
```

Comment:
Consider being defensive here and attaching a warning this is not really a generic and safe implementation of `get_info_value()` (as it won't deal with overlapping field names, etc.), and it's good only for the specific test purposes it was created for.

Reply:
Sure.  Might as well switch back to the non-generic version until we actually need another `INFO` field.